docs/install/add_packages.txt

```diff
@@ -16,7 +16,7 @@ with ``sudo -E``.
 
    The ``sudo -E`` is very important!
 
-#. Install ``there`` with ``pip``
+#. Install `there <https://github.com/Carreau/there>`_ with ``pip``

```

Comment:
Maybe add the pypi link instead? Since that links to the github of the package too?

```suggestion
#. Install `there <https://pypi.org/project/there>`_ with ``pip``

```